php-exercise1/index.php

```diff
@@ -0,0 +1,2 @@
+<?php
+require "src/control/control.php";
```

Comment:
Please review MVC pattern. I think its best practice to return views instead of controllers.

Reply:
Added autoloader